### PR TITLE
pipeline-utils: Group secret and not-secret values

### PR DESCRIPTION
**Pull request by cgwalters**
This helps keep clear what values are actually secrets, which
was helpful for me to understand things.

### CODE REVIEW

File: pipeline-utils.groovy [hunk 1]
```diff
@@ -39,27 +33,35 @@ def define_properties(timer) {
                     required: true),
         credentials(name: 'AWS_CI_ACCOUNT',
                     credentialType: 'org.jenkinsci.plugins.plaincredentials.impl.StringCredentialsImpl',
-                    description: "OpenShift AWS CI root account number.",
+                    description: "OpenShift AWS bI root account number.",
```
Comment (dm0-) [inline pipeline-utils.groovy]: Was this intentional?

**Comment by jlebon** [comment]
The division between secret and non-secret looks correct to me. LGTM with typo fix!

File: pipeline-utils.groovy (hunk 1 above)
Comment (cgwalters) [reply pipeline-utils.groovy]: Fixed :arrow_down:

**Pull request merged**